CONTRIBUTING.rst

```diff
@@ -26,38 +26,38 @@ Here are some guidelines about how you can contribute to Nikola:
 
 .. admonition:: A corollary:
 
-      Please **don't put multiple fixes/features in the same
-      branch/pull request**! In other words, if you're hacking on new feature X
-      and find a bugfix that doesn't *require* new feature X, **make a new
+      Please **don’t put multiple fixes/features in the same
+      branch/pull request**! In other words, if you’re hacking on new feature X
+      and find a bugfix that doesn’t *require* new feature X, **make a new
       distinct branch and PR** for the bugfix.
 
 * You may want to use the `Tim Pope’s Git commit messages standard
   <http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html>`_.
   It’s not necessary, but if you are doing something big, we recommend
   describing it in the commit message.
-* While working, **rebase instead of merging** (if possible).  We encourage
-  using ``git rebase`` instead of ``git merge``.  If you are using
-  ``git pull``, please run ``git config pull.rebase true`` to prevent merges
-  from happening and replace them with rebase goodness.  There is also an
-  “emergency switch” in case rebases fail and you do not know what to do:
-  ``git pull --no-rebase``.
```

Comment:
Why did you remove this part about rebasing? I've always used Nikola's CONTRIBUTING.rst file to look this up when I forgot how to change the `pull.rebase` settings, so I will definitely miss this part ;-)